### DISCUSSION TITLE
Eviter la suppression des médias ajoutés manuellement

### DISCUSSION THREAD

**Discussion by patrowawrick** [sircharlo/meeting-media-manager#126] [Ideas] [answered]
Bonjour

Je viens de découvrir cet outil vraiment utile. Nous l'avons adopté au niveau de notre assemblée. Un grand merci à l'auteur ! J'ai juste une petite suggestion. Il est parfois nécessaire d'ajout des médias au dernier moment. Dans ce cas, il est nécessaire de relancer le téléchargement des médias jw.org et par conséquent, les médias qu'on a ajoutés manuellement sont effacés. Serait-il possible d'éviter cela ou d'avoir une fenêtre d'alerte qui demande si on veut ou non effacer ces médias, avec la possibilité de les conserver ? Ce serait vraiment pratique. Merci beaucoup. Avec toute mon affection fraternelle.

Patrick W
France 

_Google Translation :
I just discovered this really useful tool. We adopted it at the level of our assembly. A big thank you to the author ! I just have a small suggestion. Sometimes it is necessary to add media at the last moment. In this case, it is necessary to restart the download of the jw.org media and therefore, the media that have been added manually are deleted. Would it be possible to avoid that or to have an alert window that asks whether or not we want to erase these media, with the possibility of keeping them? It would be really handy. Thank you so much. With all my brotherly affection._

**Comment by sircharlo** [comment, marked as answer]
Salut Patrick,
Bien heureux que vous appréciez l'appli! Je comprends ce que tu veux dire. Au lieu d'effacer tous les médias et de recommencer à chaque fois qu'on pèse sur le bouton bleu, seulement faire le nettoyage lorsqu'on arrive dans une nouvelle semaine, par exemple. Je vais penser à quelque chose pour que ce comportement soit modifié d'ici la prochaine mise à jour, dans quelques semaines.
Merci et n'hésites surtout pas si tu as d'autres suggestions/problèmes.
Ton frère Charles

**Reply by sircharlo** [reply]
Implemented in https://github.com/sircharlo/jw-meeting-media-fetcher/commit/3631fac205a5f613b9c40031041a778f5e4b51ea

**Reply by sircharlo** [reply]
Released in #129

**Comment by patrowawrick** [comment]
Merci mon cher frère !
